cmd/captplanet/run.go

```diff
@@ -101,6 +105,22 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		if runCfg.Satellite.Web.SatelliteAddr == "" {
 			runCfg.Satellite.Web.SatelliteAddr = runCfg.Satellite.Identity.Address
 		}
+
+		database, err := satellitedb.NewDB(runCfg.Satellite.DatabaseURL)
```

Comment:
storagenode would have a different database, not the satellite one

Reply:
What do you mean? This PR is just for satellite db.

Reply:
Ah nevermind my mistake